hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1025,6 +1025,10 @@ func expectedCoreConfigResourcesForHostedCluster(hcluster *hyperv1.HostedCluster
 		// additional core config resource created when image content source specified.
 		expectedCoreConfigResources += 1
 	}
+	// additional core config required for powervs platform to enable multi path.
+	if hcluster.Spec.Platform.Type == hyperv1.PowerVSPlatform {
+		expectedCoreConfigResources += 1
```

Comment:
This should be coming from the core generated payload unless there's a reason to not https://github.com/openshift/machine-config-operator/tree/6968535ff182ae863a5309558239df879359b8d6/templates/common/powervs